src/BalanceTx.purs

```diff
@@ -434,6 +434,43 @@ _redeemersTxIns = lens' \(UnattachedUnbalancedTx rec@{ redeemersTxIns }) ->
   redeemersTxIns /\
     \rdmrs -> UnattachedUnbalancedTx rec { redeemersTxIns = rdmrs }
 
+--------------------------------------------------------------------------------
+-- Setting collateral, collateral return, total collateral
+--------------------------------------------------------------------------------
+
+setCollateral
+  :: Transaction -> Utxos -> QueryM (Either BalanceTxError Transaction)
+setCollateral transaction utxos =
```

Comment:
Indicate that these are locked utxos (the naming is confusing, because it looks like a function to set a list of utxos as collateral)